patch/misc/general-packaging-5.3.y.patch

```diff
@@ -54,9 +80,9 @@ index 5858d6e44..e81b8a6fc 100644
 +dtb_packagename=linux-dtb-"$BRANCH$LOCALVERSION"
 +libc_headers_packagename=linux-libc-dev-"$BRANCH$LOCALVERSION"
```

Comment:
I do not agree this change,  linux-libc-dev has no relationship with device/branch, only linux version itself.

Reply:
more important, libc6-dev depends on linux-libc-dev, and here linux-libc-dev-${BRANCH}${LOCALVERSION} is to build, and these two packages definitely provide same thing. conflict, thus makes your package meaningless.

Reply:
https://packages.debian.org/buster/libc6-dev

Reply:
and after you change this part, another thing you need to check is: linux-libc-dev_VERSION-armXX.dev, the VERSION here, compare with debain's official release. your should make sure it always bigger than debain's or ubuntu's version number. otherwise override by official release.